Help/FAQ.rst

```diff
@@ -20,6 +21,25 @@ In short coala does two things:
   everything but the actual analyzer routine. Write just the parameters
   your custom analysis needs and the analysis part, we'll take care of the rest
 
+What is a bear ?
+----------------
+
+A bear is meant to do static analysis on source code. It can check your code
+for potential problems, calculate metrics and even provide corrections for
+your code.
+
+Each bear has its own function and supported languages. So for different
+purposes you can use different bears.
+
+Run your first bear!
+`here <https://docs.coala.io/en/latest/Users/Tutorial.html>`__.
```

Comment:
please do not use "here" as a link target.  Imagine this is a printed version.  Imagine it is being read by a blind person.  Describe the link target with words.